packages/appstore/src/components/main-title-bar/index.tsx

```diff
@@ -36,7 +36,8 @@ const MainTitleBar = () => {
 
     return (
         <React.Fragment>
-            {is_wallet_enabled && <WalletsBanner />}
+            {/* TODO: Uncomment once useWalletMigration hook is optimized for production release. */}
```

Comment:
@nijil-deriv can we put a consistent comment for all all of the changes in this PR so we make sure if we want to revert it back, we can just search for this comment

Reply:
Good point.